lib/core-integration/src/Test/Integration/Scenario/API/Shelley/Network.hs

```diff
@@ -51,7 +51,8 @@ spec = describe "SHELLEY_NETWORK" $ do
             [ expectField #decentralizationLevel (`shouldBe` d)
             , expectField #desiredPoolNumber (`shouldBe` nOpt)
             , expectField #minimumUtxoValue (`shouldBe` Quantity minUTxOValue)
-            , expectField #hardforkAt (`shouldNotBe` Nothing)
+            , expectField (#eras . #shelley) (`shouldNotBe` Nothing)
+            , expectField (#eras . #byron) (`shouldNotBe` Nothing)
```

Comment:
Integration tests can in practice be run on any shelley-based era. So simply testing that _at least_ `byron` and `shelley` are not null, should be pretty good.

Reply:
but then you will not receive information about mary and allegra, right? :thinking:

Reply:
Ok, fine, we can probably make the expectation dependent on the `LOCAL_CLUSTER_ERA` value, by storing it in the test `Context`

Reply:
Hm, seems unexpectedly tedious though… Will look more at it later.